src/dotnet/AutoRest/JsonRpc/CallerResponse.cs

```diff
@@ -32,10 +32,16 @@ public CallerResponse(string id)
         public bool SetCompleted(JToken result)
         {
             T value;
-            if (typeof(T) == typeof(bool?))
```

Comment:
should this file "CallerResponse.cs" be identical to https://github.com/Azure/autorest.common/blob/master/src/JsonRpc/CallerResponse.cs ?

Reply:
Yes

Reply:
ok, I see that we copied this method, but the file itself does not look identical, so you may just want to check on that. If it needs to be updated, we could do it in the PR.